src/components/private-route/private-route.tsx

```diff
@@ -0,0 +1,17 @@
+import {Navigate} from 'react-router-dom';
+import { AuthorizationStatus } from '../../const';
+
+type PrivateRouteProps = {
+  authStatus: AuthorizationStatus;
+  children: JSX.Element;
+}
+
+function PrivateRoute(props: PrivateRouteProps): JSX.Element {
```

Comment:
```suggestion
function PrivateRoute({authStatus, children}: PrivateRouteProps): JSX.Element {
```
сразу здесь делай деструктуризацию